wiki/Help_centre/Website/de.md

```diff
@@ -0,0 +1,106 @@
+
+1. Klicke auf den Button `Options` im Hauptmenü oder drücke `Strg` + `O`.
+2. Tippe `block` ein, um die Optionsliste nach der Sektion `In-Game-Chat` zu filtern.
+3. Klicke auf die Einstellung, die erscheint, um sie zu aktivieren.
+
+Benutzer, die nicht auf deiner Freundesliste sind, können dich nicht mehr kontaktieren.
+
+<!-- TODO: describe how to ignore others' posts and comments when https://github.com/ppy/osu-web/issues/2319 is implemented -->
+
+#### Verhindern, dass dich neue private Nachrichten aus dem Forum erreichen
+
+Gehe zu deinen [Account-Einstellungen](https://osu.ppy.sh/home/account/edit) und klicke auf `blockiere nachrichten von benutzern, die nicht auf deiner freundesliste sind` in der Sektion Privatsphäre. Private Nachrichten von Benutzern in deiner Freundesliste werden dich trotzdem erreichen.
+
+### Was ist, wenn ich nach dem Blockieren eines Nutzers immer noch Probleme mit ihm habe?
+
+**Keine Angst, unsere Support-Mitarbeiter werden dir helfen.**
+
+Falls dich ein Benutzer immer noch auf andere Weise belästigt, seine Freunde oder andere benutzt, um deine Blockierung zu umgehen, sende uns bitte eine E-Mail an [support@ppy.sh](mailto:support@ppy.sh).
+
+Sorge dafür, dass jeder Chatverlauf oder andere Beweise in der E-Mail enthalten sind und unsere Support-Mitarbeiter werden sich um deinen Fall kümmern und die Angelegenheit für dich regeln.
+
+### Was sind diese "ich!"-Seiten, die ich auf anderen Spielerprofilen sehe?
+
+**ich!-Seiten, oder Nutzerseiten, sind spezielle Profilelemente, auf die alle Spieler mit einem [Supporter-Tag](https://osu.ppy.sh/home/support) Zugriff erhalten. Sie können alles enthalten, was du möchtest — Bilder deiner Errungenschaften, deine Beatmaps, grundsätzlich alles, was du dir wünschst!**
+
+Spieler mit einem [Supporter-Tag](https://osu.ppy.sh/home/support) erhalten das Privileg, ihre eigene Seite zu bearbeiten und auf ihrem Profil für die ganze Welt sichtbar zu machen. ich!-Seiten können alles enthalten, was du möchtest, solange die [Community-Regeln](/wiki/Rules) eingehalten werden!
+
+Auch wenn dein [Supporter-Tag](https://osu.ppy.sh/home/support) abläuft, wird deine ich!-Seite **sichtbar bleiben** und du kannst den Inhalt immer noch verändern.
+
+### Die Inhalte meiner ich!-Seite fehlen!
+
+**Das kann passieren, wenn deine ich!-Seite unangebrachten Inhalt hatte, gemäß unseren [Community-Regeln](/wiki/Rules).**
+
+In solchen Fällen darfst du beim Support-Team ein Ticket öffnen, indem du eine E-Mail an [accounts@ppy.sh](mailto:accounts@ppy.sh) sendest und deine Situation mit dem Team diskutierst.
+
+Solltest du zu irgendeinem Zeitpunkt Zweifel haben, ob etwas angemessen für eine ich!-Seite ist, darfst du einem Mitglied des [globalen Moderationsteams](/wiki/People/The_Team/Global_Moderation_Team) eine Nachricht senden, mit dem was du aushängen möchtest, und sie werden dich wissen lassen, ob es in Ordnung ist.
```

Comment:
While being a bit long, this generally reads fine to me, but if you wanna make it into two sentences that's also fine.